tools/marvin/marvin/config/test_data.py

```diff
@@ -790,6 +799,26 @@
         "format": "OVA",
         "ispublic": "true"
     },
+    "CentOS7template": {
+        "displaytext": "Centos",
+        "name": "Centos",
+        "passwordenabled": False,
+        "isdynamicallyscalable":True,
+        "ostype": "CentOS 7",
+        "url": "http://10.147.28.7/templates/cenots7/Centos7.vhd",
```

Comment:
@remibergsma  This will certainly work . Any centos 7 vhd that is downloadable by CS setup should work